docs/how-to-guides/develop-a-tool/create-and-use-tool-package.md

```diff
@@ -62,6 +62,50 @@ hello_world/
     ```
     > [!Note] There's no need to update this file if you maintain the existing folder structure.
 
+### Validate custom tool package
+You can use [pf tool validate](../../reference/pf-command-reference.md#pf-tool-validate) or `pf_client.tool.validate` to verify the correctness of the customer package tool:
+
+```bash
+pf tool validate --source <your-package-name>
+```
+
+```python
```

Comment:
shall we use tab format for CLI/SDK example here?

Reply:
use tab format for cli/sdk example